include/bh_python/fill.hpp

```diff
@@ -212,6 +212,36 @@ void fill_impl(bh::detail::accumulator_traits_holder<true, const double&>,
         weight);
 }
 
+// for multi_weight
+template <class Histogram, class VArgs>
+void fill_impl(bh::detail::accumulator_traits_holder<false, boost::span<double>>,
+               Histogram& h,
+               const VArgs& vargs,
+               const weight_t& weight,
+               py::kwargs& kwargs) {
+    // weight is not used, "use" it once to suppress "unused variable" complaints by
+    // compiler
+    (void)weight;
```

Comment:
You can use https://www.boost.org/doc/libs/latest/libs/core/doc/html/core/ignore_unused.html to make it self-explanatory.